sqlgen/reflect.go

```diff
@@ -61,7 +61,7 @@ func UnbuildStruct(table *Table, strct interface{}) ([]interface{}, error) {
 		var err error
 		values[i], err = column.Descriptor.Valuer(val).Value()
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("sqlgen: serialization error for `%s`.`%s`: %v", table.Name, column.Name, err)
```

Comment:
slightly worried about printf performance (since it does reflection), not a blocker though